spec/project_spec.rb

```diff
@@ -8,32 +8,62 @@ module GemVersionCheck
     let(:lock_file) { Lockfile.new(lock_file_content("rails_app_example.lock")) }
     let(:project) do
       Dependency.any_instance.stubs(:latest_version).returns("1.0")
-      project = Project.new("toadkicker/teststrap", %w(actionpack not_existing))
+      project = Project.new("toadkicker/teststrap", options)
```

Comment:
When reading this, I was wondering where `options` came from.
I would vote for adding a default `let(:options) { {} }` above this line.